### DISCUSSION TITLE
Notifiche su Iphone con 2022.09

### DISCUSSION THREAD

**Discussion by Moxser** [indomus/forum#1920] [HUB: Home Assistant]
Ciao a tutti.

Non riesco a venirne a capo su un problema che ho dopo l'istallazione del nuovo aggiornamento HAOS 2022.09
Ho molte automazioni che mi notificano la loro esecuzione su più cellulari. 
Il mio device che prima era identificato come notify.mobile_app_iphone_di_enrico ora viene identificato cosi notify.mobile_app_iphone
quello della mia ragazza notify.mobile_app_iphone_di_elisa non c'è più.

![image](https://user-images.githubusercontent.com/72693930/190073947-0ff3a9af-7a13-4365-97e8-30129ef010d6.png)

Mi sapete dare qualche dritta in merito.

Grazie mille a tutti

**Comment by KC-inDomus** [comment]
Dall'app companion, vai sotto "impostazionI" > "companion" > prima voce e alla voce "nome del dispositivo" ripristina il vecchio nome.

**Reply by Moxser** [reply]
Ora faccio un tentativo e vedo se mi ripristina le notifiche sui vari device. Grazie mille

**Reply by Moxser** [reply]
Nulla da fare ho cambiato nome sulla app del mio IPhone ma non distingue i due cellulari come prima. Li vede tutti e due come iPhone senza distinzione per le notifiche.
Questa sera provo a cambiare il nome anche sulla app dell'altro cell vedo se qualcosa cambia.

**Reply by KC-inDomus** [reply]
sloggati dai telefoni, cancella le integrazioni, riavvia ha, riconnettiti via tel col nome giusto

**Reply by Moxser** [reply]
Ok, stasera provo tutto, Grazie mille

**Reply by Moxser** [reply]
Tutto funzionante con i tuoi consigli, Grazie mille.
Comunque non è un problema di Home assistant ma dell'aggiornamento a IOS 16 che ho fatto ieri sera, Tutti e due i device sono stati rinominati sull'app come "iPhone" e non venivano più riconosciuti separatamente.
Potrebbe essere Utile ad atri utenti.

Grazie ancora

**Reply by KC-inDomus** [reply]
abbiamo segnalato la cosa proprio ieri [sul canale telegram](https://t.me/indomus_channel): entra anche tu se non ci sei, così da rimanere aggiornato su questo tipo di dettagli

**Reply by Moxser** [reply]
Si ci sono ma è da un po' che non ci do un occhio. 
A presto
